client/App/pages/LandingPage/components/Dropdown/Dropdown.module.scss

```diff
@@ -0,0 +1,19 @@
+.dropdown {
+  display: flex;
+  justify-content: center;
+  background-color: #f6f6f6;
+  width: 100%;
+  padding: 2rem 0;
+
+  &List {
+    display: flex;
+    width: 50%;
+    flex-wrap: wrap;
+  }
+}
+
+@media (max-width: 1200px) {
```

Comment:
It's a good practice to apply styles from bottom screen width. Here should be declared style for mobiles and added media from 1200px by **min-width**